CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-cmake_minimum_required(VERSION 3.0.2)                        
-cmake_policy(SET CMP0048 NEW)
+cmake_minimum_required(VERSION 3.20)                        
```

Comment:
The CMake version 3.20 was released on in 2021. This is relatively recent. Version 3.0.2 was released in 2018 and is expected to be already present almost everywhere. Any specific reasons for updating the CMake required version ?

Reply:
Yesterday I was able to build 2decomp with 3.18, but it failed with 3.10 I don't know how far back we can go

Reply:
I think 3.16 would be OK. We should also change the reference on the CMake donload file to the branch https://github.com/2decomp-fft/2decomp-fft/tree/UpdateCMakeVersion to make sure that the main has all the necessary changes to work smootly with xcompact3d